src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -422,9 +425,7 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 			// based on ai suggestions, they can't be separated anymore and thus will
 			// be displayed as competencies entered by hand
 			aiCompetencies: [],
-			competencies: badgeClass.extension['extensions:CompetencyExtension']
-				? competencies
-				: [],
+			competencies: badgeClass.extension['extensions:CompetencyExtension'] ? competencies : [],
```

Comment:
I actually like shorter lines of code.

Reply:
Till tuesday next week you'll have a chance of editing the prettier config in `.prettierrc`to apply your preffered formatting. I do not really mind, as long as we keep it uniform